app/presenters/completions_presenter.rb

```diff
@@ -108,6 +108,10 @@ def image_name
     end
   end
 
+  def image_alt
+    I18n.t('sign_up.completed.smiling_image_alt')
```

Comment:
This presenter is used for two images, one of which doesn't really fit the description "a smiling person with a green checkmark indicating success" (see [user-signup-ial1.svg](https://github.com/18F/identity-idp/blob/15dc6d75ab665a825723594a14a8b6cf46618fc5/app/assets/images/user-signup-ial1.svg)).

I was going to suggest creating a ticket for the Identity Verification team to handle the alt text for IAL2, but realized the alt text we came up with is for the IAL2 image? Are we actually using the IAL1 image for our authentication flows? (Or maybe we should only use one image for this screen?)

Reply:
The fastest solution would be to skip the presenter.

The best solution would be to add a conditional to the presenter if we want to call the images by different names. They both vaguely feature a person smiling. The switch for IAL2 could also be a fast follow once the precise text is determined.

Reply:
@aduth UX says the code can be merged as is. They're both images of a person smiling with a green checkmark, where one has an additional image of a web browser.